pkg/sentry/kernel/task_signals.go

```diff
@@ -96,7 +96,7 @@ var defaultActions = map[linux.Signal]SignalAction{
 // 0, the default action is taken;
```

Comment:
// 0, then the default action is taken, unless the receiver is the reaper of the pid namespace, in which case the signal is ignored;

Reply:
Thanks, will update.